index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/style.css">
+  <title>Awesome Books</title>
+</head>
+<body>
+  <header>
+    <div class="top-nav">
+      <span>Awesome Books</span>
+      <nav>
+        <ul>
+          <li><a href="#" id="books-link">List</a></li>
+          <li><a href="#" id="add-book-link">Add Book Form</a></li>
+          <li><a href="#" id="contact-link">Contact</a></li>
+        </ul>
+      </nav>
+
+    </div>
+    <div class="date-container" id="date">
+        
+    </div>
+  </header>
+
+  <main>
+    <section id="books-section" >
+      <h2 class="book-title">All Awesome Books</h2>
+      <div id="books-list" class="book-stack">
+        
+      </div>
```

Comment:
- [ Required ] I think it would be better if your design for the `book list` is similar to the design on the project wireframe for this milestone. Currently, your design varies from the wireframe in the following ways.
   
    - [ ] The book list in the project wireframe is at the center of the page.
    - [ ] The individual books are listed in a zebra-stripped `list` or `table` while your design uses cards.

## Your design
![Screenshot from 2023-07-07 02-00-18](https://github.com/lerfast/Awesome-books-a-complete-website-with-navigation/assets/16392046/8c770d1d-4401-4fc7-b761-6bcf639e6382)

## Project wireframe
![awesome_books_full_website](https://github.com/lerfast/Awesome-books-a-complete-website-with-navigation/assets/16392046/9f92141a-058c-4f9f-92fe-3d80ef7f96d0)

Reply:
I did my best, 
![image](https://github.com/lerfast/Awesome-books-a-complete-website-with-navigation/assets/111823606/f4e16526-b4e4-4298-ab4c-375db06bfbad)